src/coreclr/gc/CMakeLists.txt

```diff
@@ -104,6 +104,16 @@ list(APPEND GC_SOURCES ${GC_HEADERS})
 
 convert_to_absolute_path(GC_SOURCES ${GC_SOURCES})
 
+# clrgcr is build with standalone+regions
+if (CLR_CMAKE_TARGET_ARCH_ARM64 OR CLR_CMAKE_TARGET_ARCH_AMD64)
+  add_library_clr(clrgcr SHARED ${GC_SOURCES})
+  add_dependencies(clrgcr eventing_headers)
+  target_link_libraries(clrgcr PRIVATE ${GC_LINK_LIBRARIES})
+  target_compile_definitions(clrgcr PRIVATE -DUSE_REGIONS)
+  install_clr(TARGETS clrgcr DESTINATIONS . COMPONENT runtime)
+endif (CLR_CMAKE_TARGET_ARCH_ARM64 OR CLR_CMAKE_TARGET_ARCH_AMD64)
+
+# clrgc is build with standalone+segments
```

Comment:
I am somewhat surprised we enable `VALIDATE_HEAP` for standalone builds. Does anyone know if its required and perf implications?

Reply:
checked with Maoni and this is enabled for coreclr too. Weird that this is not a default.